src/utils/utils.js

```diff
@@ -184,3 +184,15 @@ export function guid() {
   }
   return `${S4() + S4()}-${S4()}-${S4()}-${S4()}-${S4()}${S4()}${S4()}`;
 }
+
```

Comment:
Maybe you can use moment

Reply:
Thanks for your advice! I submitted a new pr and refactored this function in it.

Reply:
> Maybe you can use moment

Thanks for your advice! I submitted a new pr and refactored this function in it.